unattended.sh

```diff
@@ -25,15 +27,29 @@ print_step_begin "System analysis"
 
 # Unattended install script only support Red Hat or compatible.
 if [ \! -e /etc/redhat-release ] ; then
-  error_and_exit "This unattended installation script only supports Red Hat compatible distributions. Please check https://documentation.centreon.com/20.10/en/installation/introduction.html for alternative installation methods."
+  error_and_exit "This unattended installation script only supports Red Hat compatible distributions. Please check https://documentation.centreon.com/$CENTREON_MAJOR_VERSION/en/installation/introduction.html for alternative installation methods."
 fi
-rhrelease=`cat /etc/redhat-release`
+rhrelease=$(rpm -E %{rhel})
 case "$rhrelease" in
-  'CentOS Linux release 7.'*)
+  '7')
+    # Good to go.
+    RELEASE_RPM_URL="http://yum.centreon.com/standard/$CENTREON_MAJOR_VERSION/el7/stable/noarch/RPMS/centreon-release-$CENTREON_MAJOR_VERSION-2.el7.centos.noarch.rpm"
```

Comment:
I'm not sure about the hardcoded '-2'. A variable may help maintain the link

Reply:
It was already hardcoded like that before.
The centreon-release rpm is usually not update early after the release so it should almost never change.
Do you have an idea to avoir this ?